README.md

```diff
@@ -1,30 +1,35 @@
 # TypeScript Error Reporter Action
+
 Ensuring type-safetiness is one of the most critical responsibility for modern software developers.
 
 This Action will provide you the easiest way to run type checks and see the result for each Pull Requests.
 
 ![TypeScript Error Reporter Action](https://user-images.githubusercontent.com/8381075/78413929-a40f0680-7654-11ea-8365-0ef72fb4d6b3.png)
 
 ## Usage
+
 This Action uses [TypeScript Compiler API](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API) to satically run diagnostics over your app and shows you the result of the checks.
 
 ```yaml
-name: main
+name: Test
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [13.x]
     steps:
-      - name: Setting up Node.js v12.13.0
+      - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
       - name: Install dependencies
-        run: |
-          yarn install
```

Comment:
Scalars should be avoided if they are not needed. Many code formatters are unable to handle it well and can cause unnoticed errors.